app/views/external_authors/_external_author_navigation.html.erb

```diff
@@ -2,7 +2,7 @@
   <% if @user %>
     <li><%= span_if_current ts("External Authors"), user_external_authors_path(@user) %></li>
     <li><%= span_if_current ts("Profile"), user_profile_path(@user) %>
-  <% elsif permit?("opendoors") %>
+  <% elsif permit?("opendoors", :get_user_method => 'current_user') %>
```

Comment:
Hound doesn't run on .html.erb files, so it didn't mention that you should probably change to the `get_user_method: 'current_user'` syntax here as well.